functorrent.cabal

```diff
@@ -81,3 +91,31 @@ test-suite functorrent-test
                      QuickCheck,
                      tasty-quickcheck,
                      test-framework-quickcheck2
+
+test-suite control-thread-tests
```

Comment:
A new test suite is not required. Cant you add a few more tests to the existing quite?